### DISCUSSION TITLE
Question: mount config in s3

### DISCUSSION THREAD

**Discussion by shaunc** [directus/directus#2763] [Archive]
The docker image gives instructions for mounting `uploads` to S3. Is it also possible to mount `config` as well (eg using s3fs)? This would perhaps simplify backups, and we could use s3 versioning.

Would there be any performance issues? (I would assume config is cached and not constantly read, but thought I'd check.)

**Comment by WoLfulus** [comment]
i'm not that familiar with s3fs even though i heard about it already, but if it has a local cache for non cloud-modified files, it should be fine. 

configs are loaded for every request since they are part of the code (required by php), only some of the settings lives in the database
